components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2077,6 +2077,141 @@ void RewardsServiceImpl::SaveInlineMediaInfo(
                     std::move(callback)));
 }
 
+void RewardsServiceImpl::UpdateMediaDuration(
+    const std::string& publisher_key,
+    uint64_t duration) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->UpdateMediaDuration(publisher_key, duration);
+}
+
+void RewardsServiceImpl::GetPublisherInfo(
+    const std::string& publisher_key,
+    GetPublisherInfoCallback callback) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->GetPublisherInfo(
+      publisher_key,
+      base::BindOnce(&RewardsServiceImpl::OnPublisherInfo,
+          AsWeakPtr(),
+          std::move(callback)));
+}
+
+void RewardsServiceImpl::OnPublisherInfo(
+    GetPublisherInfoCallback callback,
+    const ledger::Result result,
+    ledger::PublisherInfoPtr info) {
+  const auto result_converted = static_cast<int>(result);
+  if (result != ledger::Result::LEDGER_OK) {
+    std::move(callback).Run(result_converted, nullptr);
+    return;
+  }
+
+  brave_rewards::PublisherInfo brave_publisher_info;
```

Comment:
let's add function to covert this like we have for content site `PublisherInfoToContentSite` (can be just renamed if we drop content site)

Reply:
Done.